pkg/plugin/plugin.go

```diff
@@ -702,23 +702,52 @@ func (e *AutoscaleEnforcer) Score(
 		return score, nil
 	}
 
-	remainingCPU := node.remainingReservableCPU()
-	remainingMem := node.remainingReservableMemSlots()
-	totalCPU := e.state.maxTotalReservableCPU
-	totalMem := e.state.maxTotalReservableMemSlots
+	cpuRemaining := node.remainingReservableCPU()
+	cpuTotal := node.totalReservableCPU()
+	memRemaining := node.remainingReservableMemSlots()
+	memTotal := node.totalReservableMemSlots()
+
+	cpuFraction := 1 - cpuRemaining.AsFloat64()/cpuTotal.AsFloat64()
+	memFraction := 1 - float64(memRemaining)/float64(memTotal)
+	cpuScale := node.totalReservableCPU().AsFloat64() / e.state.maxTotalReservableCPU.AsFloat64()
+	memScale := float64(node.totalReservableMemSlots()) / float64(e.state.maxTotalReservableMemSlots)
```

Comment:
The formatting hurts my eyes. Curse you gofmt! :rofl:

Reply:
yeah.... :sweat_smile: